### ISSUE TITLE
File manager issue when deleting folders that you are in.

### ISSUE THREAD

**Issue by rburch** [contao/contao#587]
**4.4.40**

**File manager**  

Probably not a use case that often happens but, when drilled down into a folder in the file manager, if you delete the folder you are in, Contao doesn't know where to put you when the backend refreshes. It sent me to the backend home page. When I click file manager again it tries to find the folder I was in from cache but is now deleted so I get an execute script timeout.

All you have to do is clear your backend profile cache and you're good again, but maybe what it should do is take you back to the top level of the file manager when that happens.

**Comment by leofeyer** [comment]
What exactly do you mean by "drilled down into a folder"? How can I reproduce this?

**Comment by rburch** [comment]
Sorry, yes that is a bit confusing. So say I'm in Files/subfolder1/subfolder2. I've pressed the file name of subfolder2 so I only see subfolder2 contents. If from there I delete subfolder2 Contao doesn't know what view to load after it is deleted because where I had navigated to is now gone. Does that make more sense?

**Comment by leofeyer** [comment]
Yes, now I know what you mean. 👍

**Comment by leofeyer** [comment]
I was not able to reproduce this.

* I have created a new folder `contaodemo/test`.
* I have clicked the folder name to activate the breadcrumb menu.

<img width="398" alt="" src="https://user-images.githubusercontent.com/1192057/62879313-22681280-bd2b-11e9-8a48-5e156e9bdf1b.png">

* I have clicked the "delete" icon to delete the `contaodemo/test` folder.
* The folder was removed and I was taken back to the file manager (no more breadcrumb menu).
